packages/extension-ui/src/Popup/Signing/Signing.test.tsx

```diff
@@ -0,0 +1,194 @@
+// Copyright 2019 @polkadot/extension-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+
+import { configure, mount, ReactWrapper } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import * as messaging from '@polkadot/extension-ui/messaging';
+import { AccountJson, SigningRequest } from '@polkadot/extension/background/types';
+import { flushAllPromises } from '@polkadot/extension-ui/testHelpers';
+import {
+  AccountContext,
+  ActionContext,
+  defaultTheme,
+  SigningContext,
+  Address,
+  Input,
+  Button
+} from '@polkadot/extension-ui/components';
+import { ThemeProvider } from 'styled-components';
+import Signing from '.';
+import TransactionIndex from './TransactionIndex';
+import Request from './Request';
+import Details from '@polkadot/extension-ui/Popup/Signing/Details';
+
+configure({ adapter: new Adapter() });
+
+describe('Create Account', () => {
+  let wrapper: ReactWrapper;
+  let onActionStub: jest.Mock;
+  let signRequests: SigningRequest[] = [];
+  const accounts: AccountJson[] = [{
+    address: '5D4bqjQRPgdMBK8bNvhX4tSuCtSGZS7rZjD5XH5SoKcFeKn5',
+    genesisHash: '0xdcd1346701ca8396496e52aa2785b1748deb6db09551b72159dcb3e08991025b',
+    isExternal: false,
+    name: 'acc1'
+  }, {
+    address: '5E9nq1yGJJFiP8C75ryD9J2R62q2cesz6NumLnuXRgmuN5DG',
+    genesisHash: '0xdcd1346701ca8396496e52aa2785b1748deb6db09551b72159dcb3e08991025b',
+    isExternal: false,
+    name: 'acc2'
+  }];
+  const mountComponent = (): ReactWrapper => mount(
+    <ActionContext.Provider value={onActionStub}>
+      <AccountContext.Provider value={accounts}>
+        <SigningContext.Provider value={signRequests}>
+          <ThemeProvider theme={defaultTheme}>
+            <Signing/>
+          </ThemeProvider>
+        </SigningContext.Provider>
+      </AccountContext.Provider>
+    </ActionContext.Provider>
+  );
+
+  beforeEach(async () => {
+    jest.spyOn(messaging, 'cancelSignRequest').mockResolvedValue(true);
```

Comment:
I've noticed you stub the method differently here than in `Export.test.tsx`, L30. I slightly prefer your version. So perhaps for consistency you can change it in the `Export.test.tsx`?